src/net/Lib/Clients.cs

```diff
@@ -58,6 +63,23 @@ public async Task InitializeAsync()
             TextAnalyticsClient = new TextAnalyticsClient(Config.TextAnalyticsEndpoint, credential);
         }
 
+        public Task RefreshAppConfiguration()
+        {
+            // App Config
+            ConfigurationClient = new ConfigurationClient(Config.AppConfigEndpoint, credential);
+            
+            // Image Provider
+            ImageProvider = ImageProviderFactory.Get(
+                ConfigurationClient.GetConfigurationSetting("imageProvider").Value.Value
+            );
+
+            // Bing image search settings
+            BingImageSearchTerm = ConfigurationClient.GetConfigurationSetting("BingImageSearchTerm").Value.Value;
+            BingImageSearchApiKey = ConfigurationClient.GetConfigurationSetting("BingImageSearchApiKey").Value.Value;

```

Comment:
This should come from keyvault, not app config, might, just haven't got to that point yet.